src/index.rs

```diff
@@ -333,6 +336,7 @@ impl Index {
         tx.open_table(OUTPOINT_TO_VALUE)?;
         tx.open_table(RUNE_ID_TO_RUNE_ENTRY)?;
         tx.open_table(RUNE_TO_RUNE_ID)?;
+        tx.open_table(NUMBER_TO_RUNE_INFO)?;
```

Comment:
We don't need this table since `RUNE_ID_TO_RUNE_ENTRY` is already ordered lexicographically. 

```
#[derive(Debug, PartialEq, Copy, Clone, Hash, Eq, Ord, PartialOrd)]
pub struct RuneId {
  pub height: u32,
  pub index: u16,
}
```
```
pub(super) type RuneIdValue = (u32, u16);

impl Entry for RuneId {
  type Value = RuneIdValue;

  fn load((height, index): Self::Value) -> Self {
    Self { height, index }
  }

  fn store(self) -> Self::Value {
    (self.height, self.index)
  }
}
```

As you can see the `RuneId` and `RuneIdValue` can be ordered so the table alreadu implicitly orders by the number.